src/App.jsx

```diff
@@ -14,70 +15,61 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {selectedGood !== null ? (
```

Comment:
This issue not fixed, pleache re-check
![image](https://github.com/mate-academy/react_goods-selector-js/assets/54936044/783162b3-1945-47c0-b1f2-52d9e3027532)